Polyhedron/demo/Polyhedron/Scene_c3t3_item.cpp

```diff
@@ -1005,6 +1041,24 @@ void Scene_c3t3_item::compute_elements() const
         sd == ON_ORIENTED_BOUNDARY ||
         sb != sa || sc != sa || sd != sa)
       {
+        QColor color = d->colors[cit->subdomain_index()].darker(150);
+        f_colors.push_back(color.redF());f_colors.push_back(color.greenF());f_colors.push_back(color.blueF());
+        f_colors.push_back(color.redF());f_colors.push_back(color.greenF());f_colors.push_back(color.blueF());
+        f_colors.push_back(color.redF());f_colors.push_back(color.greenF());f_colors.push_back(color.blueF());
+
+        f_colors.push_back(color.redF());f_colors.push_back(color.greenF());f_colors.push_back(color.blueF());
+        f_colors.push_back(color.redF());f_colors.push_back(color.greenF());f_colors.push_back(color.blueF());
+        f_colors.push_back(color.redF());f_colors.push_back(color.greenF());f_colors.push_back(color.blueF());
+
+        f_colors.push_back(color.redF());f_colors.push_back(color.greenF());f_colors.push_back(color.blueF());
+        f_colors.push_back(color.redF());f_colors.push_back(color.greenF());f_colors.push_back(color.blueF());
+        f_colors.push_back(color.redF());f_colors.push_back(color.greenF());f_colors.push_back(color.blueF());
+
+        f_colors.push_back(color.redF());f_colors.push_back(color.greenF());f_colors.push_back(color.blueF());
+        f_colors.push_back(color.redF());f_colors.push_back(color.greenF());f_colors.push_back(color.blueF());
+        f_colors.push_back(color.redF());f_colors.push_back(color.greenF());f_colors.push_back(color.blueF());
+
```

Comment:
I suggest using an indexed buffer for colors. The number of different colors is really small and the buffer of colors would be small. In the buffer of indices, the color of for an OpenGL vertex would be encoded just by a `GLuint`, instead of three `GL_FLOAT`.